docs/SRS-Volere/SRS.tex

```diff
@@ -964,7 +964,8 @@ \subsection{Training Requirements}
 \end{enumerate}
 
 \section{Waiting Room}
-\lips
+There are no more requirements that will be added to the system as of this
```

Comment:
in some earlier sections I just put N/A do we want to put these bigger descriptions whenever we think a section is not applicable or just put N/A?

Reply:
I think for this one specifically more detail should be provided, but it should be fine to put N/A if there's no more detail to add